scripts/tests/equipment/equipment_weapons.lua

```diff
@@ -0,0 +1,82 @@
+-----------------------------------
+-- Equipment Weapon TP Reset Tests
+-- Tests that TP is reset when equipping/unequipping weapons from various slots
+--
+-- Note: slotID vs equipSlotID distinction:
+--   - slotID: Inventory slot ID (0 = "unequip operation")
+--   - equipSlotID: Equipment slot being modified (SLOT_MAIN=0, SLOT_SUB=1, SLOT_RANGED=2, etc.)
+--
+-- Unequip flow in EquipItem():
+--   if (slotID == 0) → UnequipItem(equipSlotID) → TP reset handled in UnequipItem() line 2141
```

Comment:
Lets not have ligatures and line numbers here please

Reply:
This kind of makes it seem like large portions of this were generated by AI? Which is a whole discussion we need to have internally

Reply:
Hold on, line 2141 _of what_? It's just calling out a function name and a line number? This isn't particularly helpful. If you wanted to reference other files you could be using `/src/map/filename.cpp:linenumber` and then you _should_ get a nice hotlink in most IDEs, etc.

BUT: Line numbers aren't stable, as soon as someone makes a change this becomes stale and irrelevant

Reply:
I missed this comment completely. I don't have strong feelings about ligatures in comments, but the line part should not have gone in. It was highlighting that TP Reset is already handled in UnequipItem of charutils.cpp at that line... which yes, it's silly and makes no sense in this test file.